cosmos-hive-auth-provider/src/main/java/com/telefonica/iot/cosmos/hive/authprovider/OAuth2AuthenticationProviderImpl.java

```diff
@@ -39,7 +39,8 @@
  */
 public class OAuth2AuthenticationProviderImpl implements PasswdAuthenticationProvider {
     
-    private static final Logger LOGGER = Logger.getLogger(HttpClientFactory.class);
+    private static final Logger LOGGER = Logger.getLogger(OAuth2AuthenticationProviderImpl.class);
+    private static final OAuth2Cache CACHE = new OAuth2Cache("/home/hive/oauth2.cache");
```

Comment:
Fixed in commits https://github.com/telefonicaid/fiware-cosmos/pull/230/commits/5e575080d4ff7668e010c52db1f72f024e1ae7c5 and https://github.com/telefonicaid/fiware-cosmos/pull/230/commits/dfa819d2d4db9b21aa7547779cac775e9d69c09b